ui/apps/dashboard/src/components/navigation/index.module.less

```diff
@@ -89,3 +89,13 @@
     overflow: auto;
   }
 }
+.terminalIcon {
+  font-size: 18px;
+  margin-right: 16px;
+  cursor: pointer;
+  transition: color 0.3s;
+
+  &:hover {
+    color: #1890ff;
+  }
+}
```

Comment:
same, you should care about warning like this:
![image](https://github.com/user-attachments/assets/7080f0e1-644e-411d-8560-dba306448361)

if you find the warning, just add extra empty line at the end of the file